config/software/ruby-cleanup.rb

```diff
@@ -230,17 +230,17 @@
     end
   end
 
-  # Check for multiple versions of the `bundler` gem and fail the build if we find more than 1.
-  # Having multiple versions has burned us too many times in the past - causes warnings when
-  # invoking binaries.
-  block "Ensure only 1 copy of bundler is installed" do
-    # The bundler regex must be surrounded by double-quotes (not single) for Windows
-    # Under powershell, it would have to be escaped with a ` character, i.e. `"^bundler$`"
-    bundler = shellout!("#{install_dir}/embedded/bin/gem list \"^bundler$\"", env: env).stdout.chomp
-    if bundler.include?(",")
-      raise "Multiple copies of bundler installed, ensure only 1 remains. Output:\n" + bundler
-    end
-  end
+  # # Check for multiple versions of the `bundler` gem and fail the build if we find more than 1.
+  # # Having multiple versions has burned us too many times in the past - causes warnings when
+  # # invoking binaries.
+  # block "Ensure only 1 copy of bundler is installed" do
+  #   # The bundler regex must be surrounded by double-quotes (not single) for Windows
+  #   # Under powershell, it would have to be escaped with a ` character, i.e. `"^bundler$`"
+  #   bundler = shellout!("#{install_dir}/embedded/bin/gem list \"^bundler$\"", env: env).stdout.chomp
+  #   if bundler.include?(",")
+  #     raise "Multiple copies of bundler installed, ensure only 1 remains. Output:\n" + bundler
+  #   end
+  # end
```

Comment:
So we are not actually fixing the issue that is causing 2 versions of bundler to be installed. I am concerned that this will cause unexpected issues in the future.

Reply:
Agreed, This was meant to be a temporary solution to get past the check and test the code. Thomas has since rolled back to 2.3.7 so I am removing this too.

Reply:
Adding this change back to see if pipelines go back green stage they were.